Ginger/Ginger/Actions/ActionEditPages/WebServices/ActWebAPIEditPage.xaml

```diff
@@ -179,6 +179,10 @@
                                 <Actions:UCValueExpression x:Name="CertificatePath" Margin="0,0,0,0"/>
                                 <CheckBox x:Name="DoNotCertificateImportFile" Content="Import Certificate file to Solution folder" ToolTip="If checked the selected certificate file will be imported to ~Documents\Certificates\" HorizontalAlignment="Left" Margin="0,1,0,0" Style="{StaticResource @InputCheckBoxStyle}"/>
                             </StackPanel>
+                            <StackPanel Orientation="Vertical">
+                                <Actions:UCValueExpression x:Name="KeyFilePathTextBox" Margin="0,0,0,0"/>
+                                <CheckBox x:Name="DoNotImportKeyFile" Content="Import key file to Solution folder" ToolTip="If checked the selected certificate file will be imported to ~Documents\Certificates\" HorizontalAlignment="Left" Margin="0,1,0,0" Style="{StaticResource @InputCheckBoxStyle}"/>
```

Comment:
Check box name is confusing. Content says import and check box says do not import. 
Also follow naming convension